### PR TITLE
[SE-2764] Address a few broken links and commands.

### PR DESCRIPTION
**Pull request by Kelketek**
## [OSPR-5059](https://openedx.atlassian.net/browse/OSPR-5059)

This addresses three issues found by our team when running through the docs recently:

1. Fixes three links which no longer worked and were redirected to the marketplace.
2. Corrects a missing bullet point in an unordered list.
3. Corrects a broken command and rephrases how the command is presented to avoid ambiguity.

### Reviewers

- [X] OpenCraft Reviewer: @gabor-boros 
- [ ] edX reviewer TBD.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Kelketek! I've created [OSPR-5059](https://openedx.atlassian.net/browse/OSPR-5059) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Review by gabor-boros — APPROVED** [review]
This looks good to me. 👍

**Comment by Kelketek** [comment]
@nedbat I'm not sure the best person to ping here, but I've been told it may be you. The build on edx-documentation appears to be broken, and it doesn't appear to be from my changes.

**Comment by natabene** [comment]
@Kelketek Thank you for your contribution. @nedbat Can you review when you have a chance?

File: en_us/developers/source/architecture.rst [hunk 1]
```diff
@@ -179,9 +179,9 @@ like order work flows and coupons.
 .. _Sass: http://sass-lang.com/
 .. _Bourbon framework: http://bourbon.io/
 .. _edx.org: http://edx.org/
-.. _XBlocks: https://open.edx.org/xblocks
-.. _LTI: https://open.edx.org/learning-tools-interoperability
-.. _JS Input: https://open.edx.org/js-input
+.. _XBlocks: ../../../xblock-tutorial/en/latest/overview/introduction.html
+.. _LTI: ../../../edx-partner-course-staff/en/latest/exercises_tools/lti_component.html
+.. _JS Input: ./extending_platform/javascript.html
```
Comment (nedbat) [inline en_us/developers/source/architecture.rst]: I see what you mean about the Marketplace links.  But I'm not sure this book is even still published. Where did you find it?

Comment (nedbat) [reply en_us/developers/source/architecture.rst]: Also, I don't think relative html paths are a good way to link pages together in general.

Comment (Kelketek) [reply en_us/developers/source/architecture.rst]: If by book, you mean the group of `.rst` files this one is a part of (sorry, not too familiar with the terminology here), it's definitely published. If you google 'edx architecture', these docs are on the first page of results. Here's a link to them. https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/architecture.html

On the relative paths, I was also unsure of this method. But I also was unsure of doing a direct link to the external URL. Should I do that instead? Or is there a third way to handle this problem?

Comment (nedbat) [reply en_us/developers/source/architecture.rst]: I see, yes, it is still published, but we no longer list it at https://docs.edx.org, which was the source of my confusion.

Comment (nedbat) [reply en_us/developers/source/architecture.rst]: The best thing would be an internal Sphinx reference to the XBlock Tutorial book.  You can change it like this:

```diff
-Open edX courses are composed of units called `XBlocks`_. Anyone can write new
+Open edX courses are composed of units called :ref:`XBlocks<xblocktutorial:Open edX XBlock Tutorial>`. Anyone can write new
```

The slug "xblocktutorial" is [defined in conf.py](https://github.com/edx/edx-documentation/blob/master/shared/conf.py#L153).  The other two references can be changed similarly.

**Comment by Kelketek** [comment]
@nedbat This is ready for another look. Thanks!

**Review by nedbat — APPROVED** [review]
Very nice, thanks! And thanks for rebasing so the tests pass. :)

**Comment by openedx-webhooks** [comment]
@Kelketek 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-documentation&entry.867055334=https://github.com/edx/edx-documentation/pull/1889&entry.1484655318=https://github.com/Kelketek&entry.752974735=2020-10-15+17:22&entry.1917517419=2020-10-24+01:18&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**